package.json

```diff
@@ -141,13 +142,14 @@
     "test-suite": "run-s test-render test-query",
     "test-suite-clean": "find test/integration/{render,query}-tests -mindepth 2 -type d  -not \\( -exec test -e \"{}/style.json\" \\; \\) -print | xargs -t rm -r",
     "test-unit": "build/run-tap --reporter classic --no-coverage test/unit",
+    "test-build": "build/run-tap --no-coverage test/build/**/*.test.js",
```

Comment:
Does this get run in CI?

Reply:
Oops, good catch -- accidentally omitted the circle.yml change